src/job.rs

```diff
@@ -1023,8 +1023,9 @@ pub fn interrupt_smtp_idle(context: &Context) {
 }
 
 pub fn interrupt_imap_idle(context: &Context) {
-    info!(context, "Interrupting IMAP-IDLE...",);
+    info!(context, "Interrupting INBOX-IDLE...",);
```

Comment:
its named IMAP-idle only inside imap.rs. outside, the idles are called INBOX-idle, MVBOX-idle, SENDBOX-idle